packages/expo-splash-screen/android/src/main/java/expo/modules/splashscreen/SplashScreenWarningActivity.kt

```diff
@@ -0,0 +1,5 @@
+package expo.modules.splashscreen
+
+interface SplashScreenWarningActivity {
```

Comment:
i don't think the interface name here fits its definition, given that the interface only consists of `onSplashScreenDismissed` which could be used for a variety of purposes

Reply:
agreed!